src/js/words/solverWords.ts

```diff
@@ -0,0 +1,106 @@
+	"right": { display: "Right", description: "", words: "YES, NOTHING, READY, PRESS, NO, WAIT, WHAT, RIGHT" },
+	"sure": { display: "Sure", description: "", words: "You ARE, DONE, LIKE, YOU'RE, YOU, HOLD, UH HUH, UR, SURE" },
+	"u": { display: "U", description: "", words: "UH HUH, SURE, NEXT, WHAT?, YOU'RE, UR, UH UH, DONE, U" },
+	"uhhuh": { display: "Uh huh", description: "", words: "UH HUH" },
+	"uhuh": { display: "Uh Uh", description: "", words: "UR, U, YOU ARE, YOU'RE, NEXT, UH UH" },
+	"uhhh": { display: "Uhhh", description: "", words: "READY, NOTHING, LEFT, WHAT, OKAY, YES, RIGHT, NO, PRESS, BLANK, UHHH" },
+	"ur": { display: "Ur", description: "Left top", words: "DONE, U, UR" },
+	"wait": { display: "Wait", description: "", words: "UHHH, NO, BLANK, OKAY, YES, LEFT, FIRST, PRESS, WHAT, WAIT" },
+	"what": { display: "What", description: "", words: "UHHH, WHAT" },
+	"whatq": { display: "What?", description: "", words: "YOU, HOLD, YOU'RE, YOUR, U, DONE, UH UH, LIKE, YOU ARE, UH HUH, UR, NEXT, WHAT?" },
+	"yes": { display: "Yes", description: "Left middle", words: "OKAY, RIGHT, UHHH, MIDDLE, FIRST, WHAT, PRESS, READY, NOTHING, YES" },
+	"youare": { display: "You Are", description: "Right bottom", words: "YOUR, NEXT, LIKE, UH HUH, WHAT?, DONE, UH UH, HOLD, YOU, U, YOU'RE, SURE, UR, YOU ARE" },
+	"you": { display: "You", description: "Right middle", words: "SURE, YOU ARE, YOUR, YOU'RE, NEXT, UH HUH, UR, HOLD, WHAT?, YOU" },
+	"your": { display: "Your", description: "Right middle", words: "UH UH, YOU ARE, UH HUH, YOUR" },
+	"youre": { display: "You're", description: "Right middle", words: "YOU, YOU'RE" },
+	"c": { display: "C", description: "Right top", words: "" },
+	"cee": { display: "Cee", description: "Right bottom", words: "" },
+	"display": { display: "Display", description: "Right bottom", words: "" },
+	"holdon": { display: "Hold On", description: "Right bottom", words: "" },
+	"lead": { display: "Lead", description: "Right bottom", words: "" },
+	"led": { display: "Led", description: "Left middle", words: "" },
+	"leed": { display: "Leed", description: "Left bottom", words: "" },
+	"read": { display: "Read", description: "Right middle", words: "" },
+	"red": { display: "Red", description: "Right middle", words: "" },
+	"reed": { display: "Reed", description: "Left bottom", words: "" },
+	"says": { display: "Says", description: "Right Bottom", words: "" },
+	"see": { display: "See", description: "Right bottom", words: "" },
+	"their": { display: "Their", description: "Right middle", words: "" },
+	"there": { display: "There", description: "Right bottom", words: "" },
+	"theyare": { display: "They Are", description: "Left middle", words: "" },
+	"theyre": { display: "They're", description: "Left bottom", words: "" },
+	"space": { display: " ", description: "Left bottom", words: "" }
+};
+
+export function words_solve(): void {
+	const stage2Section = getById("stage2")
+
+	hide(stage2Section);
```

Comment:
Best to save ui updates like this until the end (and only do them if necessary).